components/script/textinput.rs

```diff
@@ -171,7 +177,7 @@ impl<T: ClipboardProvider> TextInput<T> {
             min_length: min_length,
             selection_direction: selection_direction,
         };
-        i.set_content(initial, false);
+        i.set_content(initial, &ChangeEditPoint::NoChange);
```

Comment:
@jonleighton I think we need it because of this. set_content here needs no change. All other set_content calls need change

Reply:
The callers of TextInput::new in htmltextarea.rs and htmlinputelement.rs provide an empty string, so this does not actually help us.